### PR TITLE
Add gift and info SVGs to GW product page

### PR DESCRIPTION
**Pull request by JustinPinner**
## Why are you doing this?
Following on from the copy change PR (https://github.com/guardian/support-frontend/pull/1314) this one adds a gift and an information icon. And some rough CSS to make it look ok.

This can undoubtedly be made better, too.

So far I've only tested with Chrome. I'm sure other browsers will be weird in some/many ways.

[**Trello Card**](https://trello.com/c/oWQe33mN/2119-gifting-icon-updates-for-gw-product-page)

## Monday update
Hey it's laura the purple goblin, quick update; I've addressed the PR comments and aligned the icons:

screenshot 2018-12-17 at 12 01 45 pm

Additionally, I've updated `information.svg` so the `i` is a cutout, this lets it sit comfortably inverted against dark backgrounds while keeping its original look in the checkout ty step. (because it's set over white)

## Screenshots

screen shot 2018-12-14 at 13 28 37

screen shot 2018-12-14 at 13 29 05

screen shot 2018-12-14 at 13 29 33

screen shot 2018-12-14 at 13 29 58

### CODE REVIEW

**Comment by SarahLaughton** [comment]
The only thing I'd change is to shift the icons down slightly so they're more in-line with the copy. There's a <shiftdownabit>icon</shiftdownabit> element in HTML, right? 

(You're already regretting giving me access to this, aren't you?) 😁

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 1]
```diff
@@ -113,8 +113,35 @@ const content = (
         </ProductPageTextBlock>
         <WeeklyForm />
         <ProductPageInfoChip>
-          <p>Gifting is available for quarterly and annual subscriptions</p>
-          <p>You can cancel your subscription at any time</p>
+          <span className="component-product-page-info-chip__gifting-info">
```
Comment (walaura) [inline assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: Hi, I'm sorry to appear this late, but `ProductPageInfoChip ` already takes an `icon` parameter that renders an inline SVG in the right place, in a more reusable way

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 2]
```diff
@@ -113,8 +113,35 @@ const content = (
...
+                  <g id="Gift-SVG" transform="translate(1.000000, 1.000000)">
+                    <path d="M3.50585793,4.35813426 C4.42633252,4.35813426 5.25966585,3.73575368 6.00585793,2.49099253 L3.01141,0.1 L2.263992,0.1 L1.666364,2.49099253 L2.1,2.82809616 C2.82195248,4.35818216 2.62390512,4.35813426 3.50585793,4.35813426 Z" id="Oval-Copy" />
+                    <path d="M9.805858,4.35813426 C10.7263326,4.35813426 11.5596659,3.73575368 12.305858,2.49099253 L9.511411,0.1 L8.663992,0.1 L8.066364,2.46271893 L8,2.82809616 C8.72195248,4.35818216 8.92390519,4.35813426 9.805858,4.35813426 Z" id="Oval-Copy-2" transform="translate(10.152929, 2.229067) scale(-1, 1) translate(-10.152929, -2.229067) " />
+                    <path d="M5.5,6.7 L5.68434189e-14,6.7 L5.68434189e-14,3.5 L0.5,3 L5.5,3 L5.5,6.7 Z M5.5,9.4 L5.5,17 L1,17 L0.5,16.5 L0.5,9.4 L5.5,9.4 Z M14,6.7 L8.5,6.7 L8.5,3 L13.5,3 L14,3.5 L14,6.7 Z M13.5,9.4 L13.5,16.5 L13,17 L8.5,17 L8.5,9.4 L13.5,9.4 Z" id="Combined-Shape" stroke="#FFFFFF" />
+                  </g>
+                </g>
+              </g>
+            </svg>
```
Comment (Ap0c) [inline assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: Could this live in the `components/svgs` folder?

File: assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 3]
```diff
@@ -113,8 +113,35 @@ const content = (
...
+          <span className="component-product-page-info-chip__cancel-info">
+            <svg className="component-product-page-info-chip__cancel-info-svg" version="1.1">
+              <g id="Information-SVG" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+                <g id="Group-4">
+                  <circle id="Oval" fill="#FFFFFF" fillRule="nonzero" cx="8" cy="8" r="8" />
+                  <path d="M8.79121082,4.84545455 C7.91535223,4.84545455 7.44373607,4.2660404 7.44373607,3.63272727 C7.44373607,2.86466667 8.09052395,2.42 8.79121082,2.42 C9.6940189,2.42 10.1521603,2.98593939 10.1521603,3.64620202 C10.1521603,4.41426263 9.50537243,4.84545455 8.79121082,4.84545455 Z M7.0260189,13.092 C5.9480391,13.092 5.6650694,12.5799596 5.89414011,11.5693535 L7.05296839,6.55674747 L6.01541284,6.58369697 L6.13668556,5.8830101 C6.91822092,5.64046465 8.00967546,5.41139394 8.65646334,5.41139394 C9.46494819,5.41139394 9.70749364,5.81563636 9.50537243,6.65107071 L8.26569566,12.0005455 L9.50537243,11.9466465 L9.30325122,12.6203838 C8.68341284,12.9168283 7.80755425,13.092 7.0260189,13.092 Z" id="i" fill="#052962" />
+                </g>
+              </g>
+            </svg>
```
Comment (Ap0c) [inline assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: Could `information.jsx` be used here @Amohkhan ?

File: assets/components/svgs/gift.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import React from 'react';
+
+export default function SvgInformation() {
```
Comment (akash1810) [inline assets/components/svgs/gift.jsx]:
```suggestion
export default function SvgGift() {
```

?

Comment (walaura) [reply assets/components/svgs/gift.jsx]: ah, my bad!

**Pull request merged**